build/pipelines/templates/build-single-architecture.yaml

```diff
@@ -33,14 +27,21 @@ jobs:
       ManifestFileName: 'Package.appxmanifest'
     ${{ if eq(parameters.useReleaseAppxManifest, true) }}:
       ManifestFileName: 'Package.Release.appxmanifest'
+  templateContext:
+    sdl:
+      binskim:
+        analyzeTargetGlob: +:f|$(Agent.BuildDirectory)\binskim\**\*.dll;+:f|$(Agent.BuildDirectory)\binskim\**\*.exe
+
+    outputs:
+    - output: pipelineArtifact
+      displayName: Publish drop artifact
+      targetPath: $(Build.BinariesDirectory)\$(BuildConfiguration)\${{ parameters.platform }}
+      artifactName: drop-${{ parameters.platform }}
+
   steps:
   - checkout: self
     fetchDepth: 1
 
-  - ${{ if eq(parameters.isOSSBuild, true) }}:
-    - task: nuget-security-analysis@0
```

Comment:
After removing this, do we have any similar task to check the nuget security in pipeline now?

Reply:
Good catch. After running a test, I found that 1ES PT doesn't help us inject this task.
Added back.